### DISCUSSION TITLE
COLMAP SfM Data Import

### DISCUSSION THREAD

**Discussion by tworkool** [alicevision/AliceVision#1696] [Ideas]
Hey,
Today I wanted to try and create a dense reconstruction from a custom COLMAP pipeline sparse reconstruction result. COLMAP offers an export tool, which looks like this:

```
I0329 12:14:41.346447   147 option_manager.cc:849] COLMAP 3.9.1 (Commit e9903641 on 2024-01-08 with CUDA)
I0329 12:14:41.347059   147 option_manager.cc:851] Options can either be specified via command-line or by defining them in a .ini project file passed to `--project_path`.
  -h [ --help ]
  --random_seed arg (=0)
  --log_to_stderr arg (=1)
  --log_level arg (=0)
  --project_path arg
  --input_path arg
  --output_path arg
  --output_type arg          {BIN, TXT, NVM, Bundler, VRML, PLY, R3D, CAM}
  --skip_distortion arg (=0)
```

As you can see it does not support Meshrooms .abc format for SfM Data.

It would be very nice, I thought, if meshroom supported COLMAP sparse reconstructions out of the box or at least supported the shown formats for SfM Data.
I am not too familiar with Meshroom though and maybe such importer already exists. In that case any tip would be helpful. Thanks!

**Comment by natowi** [comment]
We have something, but it is not yet ready for release.
